ATGMediaBrowser/MediaBrowser/MediaBrowserViewController.swift

```diff
@@ -259,6 +271,12 @@ public class MediaBrowserViewController: UIViewController {
             static let tintColor: UIColor = .lightGray
             static let selectedTintColor: UIColor = .white
         }
+
+        enum Title {
+            static let top: CGFloat = 16.0
```

Comment:
Let's make this configurable using a TitleStyle structure. Having different requirements for title is a valid case. Let's try to include font also in the same